WooCommerce/Classes/Authentication/Navigation Exceptions/ULAccountMismatchViewController.swift

```diff
@@ -0,0 +1,228 @@
+import UIKit
+import WordPressAuthenticator
+import SafariServices
+
+
+/// UI presenting errors in the Unified Login flow.
+/// This view controller can either be presented from within WooCommerce
+/// or be injected into WordPressAuthenticator.
+final class ULAccountMismatchViewController: UIViewController {
+    /// The view model providing configuration for this view controller
+    /// and support for user actions
+    private let viewModel: ULAccountMismatchViewModel
+
+    /// Header View: Displays all of the Account Details
+    ///
+    private let accountHeaderView: AccountHeaderView = {
+        return AccountHeaderView.instantiateFromNib()
+    }()
+
+    @IBOutlet private weak var gravatarImageView: CircularImageView!
+    @IBOutlet private weak var userNameLabel: UILabel!
+    @IBOutlet private weak var singedInAsLabel: UILabel!
+    @IBOutlet private weak var wrongAccountLabel: UILabel!
+    @IBOutlet private weak var logOutButton: UIButton!
+    @IBOutlet private weak var primaryButton: NUXButton!
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var errorMessage: UILabel!
+    @IBOutlet private weak var extraInfoButton: UIButton!
+
+    /// Constraints on the view containing the action buttons
+    /// and the stack view containing the image and error text
+    /// Used to adjust the button width in unified views provided by WPAuthenticator
+    @IBOutlet private weak var buttonViewLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var buttonViewTrailingConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var stackViewLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var stackViewTrailingConstraint: NSLayoutConstraint!
+
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        UIDevice.isPad() ? .all : .portrait
```

Comment:
TIL! I thought we allowed all orientations all the time!

Reply:
Not for login. WordPressAuthenticator doesn't, so we can't allow rotation in the custom UI we insert into it